cluster/addons/dns/kube-dns.yaml.base

```diff
@@ -84,6 +84,17 @@ spec:
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 1
```

Comment:
We should use 100 here (the highest value).

Reply:
Ah! Sorry, I forgot. I corrected it.